samples/backend/csharp/Controllers/ChatController.cs

```diff
@@ -19,6 +20,78 @@ public ChatController(ISemanticKernelApp semanticKernelApp)
         _semanticKernelApp = semanticKernelApp;
     }
 
+    private (int MessageIndex, int FileIndex, IFormFile File) GetPosition(IFormFile formFile)
+    {
+        const string pattern = @"messages\[(\d+)\]\.files\[(\d+)\]";
```

Comment:
It'd be nice to switch this to using the regex source generator, e.g. add a method like this:
```C#
[GeneratedRegex(@"messages\[(\d+)\]\.files\[(\d+)\]")]
private static partial Regex MessagesFilesRegex();
```
and then below do:
```C#
var match = MessagesFilesRegex().Match(formFile.Name);
```

Reply:
updated, thanks!